### DISCUSSION TITLE
is this built by Golang in Backend?

### DISCUSSION THREAD

**Discussion by web1monster** [roadmap2024/Go-React-TodoList#9] [Q&A] [answered]
let me check it out.

**Comment by lucky-elsa** [comment, marked as answer]
Yes, I built this project using React for frontend and Golang for backend development.

**Comment by web1monster** [comment]
Good, But there is something missing in Create Function.

**Comment by lucky-elsa** [comment]
Really?
I see.
I will fix that soon.
